doc.go

```diff
@@ -33,12 +33,19 @@ Note that in the SQLair `db` tags (i.e. the column names) appear in the input/ou
 
 # Syntax
 
-SQLair input expressions take the format:
+The SQLair expressions specify Go values to use as query inputs or outputs. The
+input expressions can take values from structs maps and slices whereas the
```

Comment:
```suggestion
input expressions can take values from structs, maps and slices whereas the
```

Additionally, should this and the next sentence be `or` or `and`?

Reply:
Added the comma, I've changed the `whereas` to `while` which I think fits a bit better. There are already a few `and`s in the sentence.